src/dotnet/Program.cs

```diff
@@ -178,6 +182,9 @@ private static void ConfigureDotNetForFirstTimeUse()
                             environmentProvider);
 
                         dotnetConfigurer.Configure();
+
+                        if (!File.Exists(Telemetry.TelemetrySentinel))
+                            File.Create(Telemetry.TelemetrySentinel);
```

Comment:
>  if the user sets the SKIP_FIRST_RUN flag, he won't get the EULA and we will still create the telemetry sentinel, right?

Yes that's right, i think the correct way will be to have eula and it's sentinel and then a separate nuget cache sentinel. For now, i am reusing the nuget cahe sentinel for all purposes, we can rethink this design for preview3